shared_model/backend/protobuf/commands/proto_create_domain.hpp

```diff
@@ -46,7 +45,8 @@ namespace shared_model {
       }
 
      private:
-      const iroha::protocol::CreateDomain &create_domain_;
+      const iroha::protocol::CreateDomain &create_domain_{
+          proto_->create_domain()};
```

Comment:
Is it formatted right?

Reply:
Maybe not. I will double  check formatting.